examples/images_contours_and_fields/shading_example.py

```diff
@@ -3,19 +3,17 @@
 Shading Example
 ===============
 
+Example showing how to make shaded relief plots
+like Mathematica
+(http://reference.wolfram.com/mathematica/ref/ReliefPlot.html)
+or Generic Mapping Tools
+(http://gmt.soest.hawaii.edu/gmt/doc/gmt/html/GMT_Docs/node145.html)
 """
 import numpy as np
 import matplotlib.pyplot as plt
 from matplotlib.colors import LightSource
 from matplotlib.cbook import get_sample_data
 
-# Example showing how to make shaded relief plots
-# like Mathematica
-# (http://reference.wolfram.com/mathematica/ref/ReliefPlot.html)
-# or Generic Mapping Tools
-# (http://gmt.soest.hawaii.edu/gmt/doc/gmt/html/GMT_Docs/node145.html)
-
-
```

Comment:
PEP8 requires there to be two empty lines before `def main():` on the line below my comment.